lerobot/scripts/train.py

```diff
@@ -384,7 +418,7 @@ def evaluate_and_checkpoint_if_needed(step):
     else:
         shuffle = True
         sampler = None
-    dataloader = torch.utils.data.DataLoader(
+    dataloader = MultiEpochsDataLoader(
```

Comment:
There is a dataloader for online training as well. We might consider putting this there too, or not. I'm not sure it will be compatible because we need to keep updating the sampler weights throughout training. Maybe that's out of scope for this PR.

So my ask: please consider putting a comment near where the other `dataloader` is created with:
`# MultiEpochsDataLoader is not used here because it has not been verified to be compatible with the way we update the dataset and sampler throughout training.`

That might protect us from someone changing it without thinking about it carefully.